engine/src/core/debug/logger/logger.h

```diff
@@ -0,0 +1,93 @@
+//
```

Comment:
Fix naming in all files -> our convention is that methods start with capital letters.

Reply:
ehh this naming is to provide API simular to spdlog since we want to use it when they will support g++-9